.buildkite/load-testing.yml

```diff
@@ -0,0 +1,3 @@
+steps:
```

Comment:
This is a placeholder I assume? Could you add a comment explaining this, so that no one deletes this file in the future.

Reply:
I will actually work on this, right after this PR is merged.